sip/sip-005-blocks-and-transactions.md

```diff
@@ -625,14 +625,25 @@ The _payload type ID_ can take any of the following values:
 * `0x04`:  the payload that follows is a **coinbase payload**.
 
 The _STX token-transfer_ structure is encoded as follows:
-* A **recipient address**, comprised of a 1-byte address version number and a
-  20-byte public key hash that identifies a (possibly unmaterialized) standard
-account to recieve the tokens,
+* A **recipient principal** encoded as follows:
+  * A 1-byte type field indicating whether the principal is
+    * `0x05`: a recipient address
+    * `0x06`: a contract recipient
+  * If a simple recipient address, the 1-byte type is followed by a
+    1-byte address version number and a 20-byte hash identifying a standard
+    recipient account.
+  * If a contract recipient address, the 1-byte type is followed by
+    the issuer address of the contract, encoded with a 1-byte address
+    version number and the 20-byte hash that identifies the standard
+    account of the issuer. This is followed by the encoding of the
+    contract name -- encoded as described above.
 * An 8-byte number denominating the number of microSTX to send to the recipient
   address's account.
 
 Note that if a transaction contains a token-transfer payload, it MUST have only
-a standard authorization field.  It cannot be sponsored.
+a standard authorization field. It cannot be sponsored. The recipient principal
+does not need to be a materialized account -- STX may be transfered to an account
+which has not been used in any prior transactions.
```

Comment:
This documentation should clarify what happens if you send STX to a contract account that has not yet been materialized.